Framework/Indexing/src/IndexInfo.cpp

```diff
@@ -90,7 +90,8 @@ IndexInfo::~IndexInfo() = default;
 
 IndexInfo &IndexInfo::operator=(const IndexInfo &other) {
   auto copy(other);
-  return *this = std::move(copy);
+  *this = std::move(copy);
+  return *this;
```

Comment:
Isn't the old version doing the same thing?

Reply:
Functionally they are the same yes. Clang Tidy doesn't see that though. This is arguably more readable at a quick glance.